app/scripts/lib/experiments/grouping-rules/signup-password-confirm.js

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const BaseGroupingRule = require('./base');
+
+  module.exports = class SignupPasswordConfirmGroupingRule extends BaseGroupingRule {
+    constructor () {
+      super();
+      this.name = 'signupPasswordConfirm';
+    }
+
+    choose (subject = {}) {
+      const sampleRate = SignupPasswordConfirmGroupingRule.sampleRate(subject.env);
+
+      return !! subject.uniqueUserId && this.bernoulliTrial(sampleRate, subject.uniqueUserId);
```

Comment:
One observation from previous experiments: having "treatment" and "control" groups that are the same size tends to make exploratory analysis easier, as it makes it easier to eyeball the data for oddities.  There may be other data-analysis reasons why it's useful to have an explicitly-labelled control group, but I certainly appreciated this one while analyzing the send-sms results.